apps/cache-manager/src/app.ts

```diff
@@ -41,6 +49,7 @@ app.post('/:port/prerender', async (req, res) => {
       component = response.data
     }
 
+    res.set(headers)
```

Comment:
Do we need to send back content language?

Reply:
we have a task for it in the project